### ISSUE TITLE
Sage tests failing due to repository not found

### ISSUE THREAD

**Issue by oscarbenjamin** [sympy/sympy#24662]
CC @mkoeppe 

After merging #24661 I see failures of the Sage job:
```
  remote: Repository not found.
  Error: fatal: repository 'https://github.com/sagemath/sage/' not found
  Error: The process '/usr/bin/git' failed with exit code 128
```
Is this due to the migration of Sage to GitHub?

failure log (I tried rerunning this but same result):
https://github.com/sympy/sympy/actions/runs/4098648356/jobs/7067954082

**Comment by oscarbenjamin** [comment]
Currently `https://github.com/sagemath/sage` gives a 404.

**Comment by mkoeppe** [comment]
Yes, we are near the end of our migration to GitHub. Repo is planned to be accessible again by tomorrow.

**Comment by oscarbenjamin** [comment]
Thank you.

**Comment by oscarbenjamin** [comment]
The Sage job is running again.

There are failures in calculus.py to be fixed in #24658

There are also failures in
```
src/sage/schemes/elliptic_curves/ell_number_field.py  # 2 doctests failed
```
I'm not sure if those are related to sympy.

I wonder if this should be changed now that Sage has moved to GitHub:
https://github.com/sympy/sympy/blob/43eed48717b638272a9b8269cde20f460a6b9d6c/.github/workflows/ci-sage.yml#L57

**Comment by mkoeppe** [comment]
The workflow can be simplified now. I'll send a PR shortly.

**Comment by oscarbenjamin** [comment]
There are some doctest failures now in the Sage job e.g.:
```
File "src/sage/calculus/calculus.py", line 1693, in sage.calculus.calculus.laplace
Failed example:
    laplace(t^n, t, s, algorithm='sympy')
Expected:
    (s^(-n - 1)*gamma(n + 1), 0, re(n) > -1)
Got:
    (gamma(n + 1)/(s*s^n), 0, re(n) > -1)
```
Previously in trac there was a specific ticket tracking updating the SymPy version and the Sage job running in SymPy CI tracked that ticket so I could send a fix to that ticket.

How exactly would I update those doctests in Sage's new workflow?

Is it just a PR to the main branch in the Sage repo or does it need to be directed at a specific other branch that tracks updating Sage's SymPy version?

**Comment by mkoeppe** [comment]
It would probably be best if you could open a PR against the `develop` branch, based on the branch of https://github.com/sagemath/sage/issues/34489

I'll then send the promised PR that replaces the reference to the Trac ticket by that PR branch.

**Comment by oscarbenjamin** [comment]
> based on the branch of [sagemath/sage#34489](https://github.com/sagemath/sage/issues/34489)

I'm not sure how to get that branch. I tried adding `https://github.com/sagemath/sagetrac-mirror.git` as a remote and also cloning it but both just seem to stall.
